### PR TITLE
PROD-5

### PR DESCRIPTION
**Pull request by MarcinkEve**
[PROD-2](https://prodevshub.atlassian.net/browse/PROD-2)

[PROD-5](https://prodevshub.atlassian.net/browse/PROD-5)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,22 @@
 .App {
   text-align: center;
 }
```
Comment (prodevshub) [inline src/App.css]: This file shouldn't be changed.  They should be exact as it is in main branch. Please reverse them to original.

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import "./App.css";
```
Comment (prodevshub) [inline src/App.jsx]: This file shouldn't be changed.  They should be exact as it is in main branch. Please reverse them to original.

**Comment by prodevshub** [comment]
You need to resolve conflicts in order to merge this PR.

**Review by prodevshub — CHANGES_REQUESTED** [review]
Please resolve issues regarding this PR by resolving conflicts with main branch

**Review by prodevshub — APPROVED** [review]

**Review by SwallowEve — APPROVED** [review]
Nice done, Evelina!

**Review by INKR1 — APPROVED** [review]
Good job :)

**Pull request merged**